--- a/src/RooUnfoldTH1Helpers.cxx
+++ b/src/RooUnfoldTH1Helpers.cxx
@@ -212,6 +212,8 @@ Hist *createHist(const TVectorD &vec, const TVectorD &errvec, const char *name,
 
 // Specializations for TH1 as both Hist and AnyHist
 template TH1 *createHist<TH1, TH1>(const char *name, const char *title, const std::vector<Variable<TH1>> &x);
+template TH1 *createHist<TH1, TH1>(const TVectorD &vec, const char *name, const char *title,
+                                   const std::vector<Variable<TH1>> &x, bool overflow);
 template TH1 *createHist<TH1, TH1>(const TVectorD &vec, const TVectorD &errvec, const char *name, const char *title,
                                    const std::vector<Variable<TH1>> &x, bool overflow);
 
